src/components/landing/ScrollIndicator.tsx

```diff
@@ -0,0 +1,34 @@
+import { motion } from 'framer-motion';
+
+export const ScrollIndicator = () => {
+  const handleClick = () => {
+    const targetElement = document.getElementById('what-is-section');
+    if (targetElement) {
+      targetElement.scrollIntoView({
+        behavior: 'smooth',
+        block: 'start',
+      });
+    }
+  };
```

Comment:
this component does not handle cases where `document.getElementById('what-is-section')` returns `null`. this might lead to runtime errors.

Reply:
On line no 7 we have a `if` case right? in case of element is not found it wont scroll.